packages/common/src/utils/stream-utils.ts

```diff
@@ -78,6 +78,10 @@ export class StreamUtils {
   static deserializeCommit(commit: any): any {
     const cloned = cloneDeep(commit)
 
+    if (cloned.header?.model) {
```

Comment:
Yep, there's an early return here https://github.com/ceramicnetwork/js-ceramic/blob/develop/packages/common/src/utils/stream-utils.ts#L48. Without that, the model doesn't get converted from a string.

Reply:
But that's only when dealing with a signed commit envelope, which shouldn't have a model field anyway since the commit metadata will be inside the payload, not the envelope.
Have you seen this actually cause an issue?  I'd be curious to see a repro if there's a case where a SignedCommitContainer actually has a `metadata.model` field.

Reply:
Yep, this was the fix I needed for the rust code when passing in a model id that was a stream id. There's portions of the code that use this model field, even if it is a signed commit container (`jws` field present).

Reply:
that's surprising to me.  Do you have a repro that creates/shows a commit envelope that also has a model?

Reply:
This test should fail if that line is in the original place https://github.com/3box/composedb-rs/blob/main/lib/src/ceramic/mod.rs#L323

Reply:
And pretty sure if the block header omits the model, the test also fails since model is not present.